bin/run-wer-automation.sh

```diff
@@ -32,7 +32,7 @@ export ds_validation_step
 ds_dropout_rate=0.05
 export ds_dropout_rate
 
-ds_default_stddev=0.1
+ds_default_stddev=0.046875
```

Comment:
I can remove it, but I've kept `run-wer-automation.sh` rather explicit. So one can see, more-or-less, what's going on without looking at DeepSpeech.ipynb.